### PR TITLE
#159053948 Creates  a sample index page using Materialize CSS framework

### PR DESCRIPTION
**Pull request by johnmusiu**
#### What does this PR do?
Creates `index.html` page

#### Description of tasks to be completed
Use Materialize CSS framework (our chosen framework) to create a simple landing page.

#### How should this be manually tested?
Clone the repository branch using the link [https://github.com/andela/ah-titans/tree/ch-adds-indexpage-using-materialize-159053948]

Navigate to the project folder path `ah-titans/authors/templates/`

Open `index.html` using an updated browser of your choice.

#### What are the relevant Pivotal Tracker Stories?
[#159053948](https://www.pivotaltracker.com/story/show/159053948)

#### Screenshot
image

### CODE REVIEW

**Comment by janetnim** [comment]
indent the divs under div class="card blue-grey darken-1"

File: authors/templates/index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+                    <p>Welcome to the Authors Haven by team Titans.</p>
+                </div>
+                <div class="card-action">
+                    Site is Under Development
+                </div>
+                </div>
+            </div>
+        </div>
+   </body>
+</html>
```
Comment (petermwash) [inline authors/templates/index.html]: You should add an end of a line

**Comment by dcDalin** [comment]
A screenshot of the live template on the PR would be nice.

File: authors/templates/index.html (hunk 1 above)
Comment (johnmusiu) [reply authors/templates/index.html]: Is this a formatting specification for HTML files?

**Pull request merged**